Public/Src/Engine/Processes/ReportedFileOperation.cs

```diff
@@ -172,6 +172,13 @@ public enum ReportedFileOperation : byte
         /// </summary>
         ChangedReadWriteToReadAccess,
 
+        /// <summary>
+        /// This is a quazy operation. We issue this only when <see cref="FileAccessPolicy.OverrideAllowWriteForExistingFiles"/> is set, representing

```

Comment:
> We issue this
Is the sandbox expected to send this report, or is it the BuildXL engine that figures this out based the "standard" reports from the sandbox?  Please clarify this in the comment

Reply:
Sandbox. Will change

---
In reply to: [299162179](https://github.com/microsoft/BuildXL/pull/536#discussion_r299162179) [](ancestors = 299162179)